widgets/selectivity/selectivity.js

```diff
@@ -3372,8 +3372,12 @@ var callSuper = Selectivity.inherits(MultipleInput, Selectivity, {
     _updateInputWidth: function() {
         var inputContent =
             this.input.value || (!this._data.length && this.options.placeholder) || '';
-        this.input.setAttribute('size', inputContent.length + 2);
-
+        const placeholderLength= typeof this.options.placeholder!== 'undefined' && this.options.placeholder.length;
```

Comment:
Took reference from here for checking `undefined` string.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined